src/17A70.c

```diff
@@ -406,24 +421,25 @@ void Intro_Tick(void) {
 
 char D_800C8F68[] = "PRESS START";
 char D_800C8F74[] = "@1997 TREASURE/ENIX";
-char D_800C8F88[] = "LICENED BY NINTENDO";
+char D_800C8F88[] = "LICENSED TO NINTENDO";
 
 // prints "Press start" and copyright info
-void func_80017F08(void) {
+void Title_Copyright(void) {
     func_80017770();
-    func_800276DC(0x39U, &D_800C8F68, 0xFFCA, 0xFFE4, 0, func_80027588(0U, (0x1F - D_801781A0 / 4), (0x1F - D_801781A0 / 4), 0x1FU));
-    func_800276DC(0x49U, &D_800C8F74, 0xFFAA, 0xFFC0, 0, func_80027588(2U, 0x1FU, 0x1FU, 0x18U));
-    func_800276DC(0x60U, &D_800C8F88, 0xFFA6, 0xFFAE, 0, func_80027588(2U, 0x1FU, 0x1FU, 0x18U));
+    Text_PrintASCII(0x39U, &D_800C8F68, 0xFFCA, 0xFFE4, 0, Text_ConvertColor(0U, (0x1F - D_801781A0[0] / 4), (0x1F - D_801781A0[0] / 4), 0x1FU));
+    Text_PrintASCII(0x49U, &D_800C8F74, 0xFFAA, 0xFFC0, 0, Text_ConvertColor(2U, 0x1FU, 0x1FU, 0x18U));
+    Text_PrintASCII(0x60U, &D_800C8F88, 0xFFA6, 0xFFAE, 0, Text_ConvertColor(2U, 0x1FU, 0x1FU, 0x18U));
```

Comment:
if the Us next to the hex here aren't required for the OK, remove them for readability.